### DISCUSSION TITLE
How do I add/update config?

### DISCUSSION THREAD

**Discussion by heejit** [rtCamp/Frappe-Manager#203] [General]
Thanks for this tool it is really easy and very usefull.

I try many instruction to install ERPNext but there is always a problem only this tool work for me without ton
of installation instruction and running various commands.

how do we add/update config of nginx or bench in already created site.

**Comment by Xieyt** [comment]
Thank you for your feedback and appreciation. We're glad that our tool has made the ERPNext installation process easier for you.

Regarding add/update config of nginx:
> The `default.conf` file is located in the `~/frappe/sites/<sitename>/configs/nginx/conf.d` directory.
- Currently, users can override some of the directives defined in `default.conf` or add new directives using the `custom` directory. This functionality is enabled by the include directive mentioned [here](https://github.com/rtCamp/Frappe-Manager/blob/57d7bf0c1da6304eb8fd8ab0128bf7d0ee07b611/Docker/nginx/template.conf#L132).
- Changes made to `default.conf` do not persist because `default.conf` is recreated whenever the container entrypoint runs.

To fix the shortcomings of current approach I have created Issue regarding the above [here](https://github.com/rtCamp/Frappe-Manager/issues/205).

**Comment by heejit** [comment]
Thanks
